src/extremeweatherbench/evaluate.py

```diff
@@ -253,11 +398,15 @@ def _open_obs_datasets(eval_config: config.Config):
     """Open the observation datasets specified for evaluation."""
     point_obs = None
     gridded_obs = None
+    if eval_config.point_obs_path:
+        point_obs = pd.read_parquet(
+            eval_config.point_obs_path, storage_options=POINT_OBS_STORAGE_OPTIONS
+        )
     if eval_config.gridded_obs_path:
         gridded_obs = xr.open_zarr(
             eval_config.gridded_obs_path,
             chunks=None,
-            storage_options=dict(token="anon"),
+            storage_options=GRIDDED_OBS_STORAGE_OPTIONS,
```

Comment:
This circles back to my comment at the top of the file about your global variables. This is a tough call; the "best" way would be to provide a set of default options, and then merge it with user-passed ones. Hard-coded behavior, like the auth mode, will lead to non-obvious failure modes for users down the line, without an easy way to rectify without hacking the library. It _might_ be a good idea to go ahead and provide helper arguments to this method or package within the `config.Config` object the ability for a user to over-ride these storage_options.

If you want to hard-code everything, leave a description in the docstring about this behavior so that at least it's very explicit what is happening.